src/content/developers/docs/data-availability.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Data availability 
+description: An overview of problems and solutions relating to data availability in Ethereum 
+lang: en
+sidebar: true
+---
+
+In Ethereum, a node proposing a new block is required to make the underlying data available to the remaining peer-to-peer nodes on the network. This concept, known as data availability, is fundamental to the security and functionality of every blockchain, including Ethereum. Data availability is so important that an entire field of research studying the so-called “data availability problem” exists. 
+
+## What is data availability? {#what-is-data-availability}
+
+“Data availability” is a guarantee that all transaction data for a block was published by the block proposer and is available to other network participants. As explained in “[Intro to Ethereum](/developers/docs/intro-to-ethereum/)”, Ethereum transactions are processed in [blocks](/developers/docs/blocks/) and chained together to form the “blockchain”.
```

Comment:
This feels like a very roundabout way of saying that all data should be available for transactions to reproduce the current world state

Reply:
Maybe you can explain this with more detail? Not totally sure what you mean here.